tests/ContextTest.py

```diff
@@ -1,5 +1,6 @@
+import rdflib
 from rdflib.term import URIRef
-from rdflib.graph import ConjunctiveGraph
+from unittest.mock import MagicMock
```

Comment:
This won't work in Python 2.7, please fall back to `mock` (already included in setup.py).